marsha/core/views.py

```diff
@@ -64,7 +66,7 @@ def get_context_data(self):
         else:
```

Comment:
```
except LTIException:
    return {"state": "error"}
```

We need an explicit JSON with `null` for the video here, otherwise get just get `<div ... data-video />`

Reply:
Done (see above as this comment is about line 57)